### ISSUE TITLE
Accessibility: need HTML structure in chat window in web version 

### ISSUE THREAD

**Issue by ssawczyn** [jitsi/jitsi-meet#5789]
As a screen reader user, I want to be able to quickly navigate through chat messages.  Unfortunately, because the HTML of the chat window does not contain semantic hierarchy, I cannot quickly navigate using my screen reader.  Screen readers allow the user to quickly navigate content when semantic structure (headings, lists, landmark regions) is in place making the experience far more efficient for the user.

Consider:

- Implementing chat messages as an unordered list ,UL> with an <LI> containing each individual message.  The LI should also contain the time stamp and sender of each message.
- Implement headings or landmark regions to identify secants of the chat window:  chat history, emojis, any other areas that would constitute a specific section of the chat UI.
-

**Comment by Echolon** [comment]
Hi,
The team is aware of accessibility. Any reports are now grouped in the pinned issue #6090.

---
*I'm a community contributor and not directly affiliated with the Jitsi team.*

**Comment by missmatsuko** [comment]
@Echolon This issue is one of the ones referenced in #6090. I don't think it should be closed until it's actually addressed, then #6090 can be closed when all the referenced issues are closed. That makes more sense to me, unless someone directed you to do this?

**Comment by Echolon** [comment]
Will keep it open then.

But its not deleted. We are trying to condense the information to a more practical way.
